src/app/components/navigation/navigation.component.html

```diff
@@ -21,8 +21,11 @@
           </li>
         </ul>
 
-      <div class="lang">
-        <span class="icon"><i class="fa fa-language" aria-hidden="true"></i></span>
+      <div class="lang" title="{{'Change interface language'|translate}}">
```

Comment:
Isn't "Change language" enough? Or "Select language"?

`src/assets/i18n/en.json` (and all other language files) should be updated too, shouldn't it, with the new string to translate?

I will rebuild and test.

Reply:
> Isn't "Change language" enough? Or "Select language"?

This is to explain the purpose of the field, I think it's better to be specific. Could you expand why you think "interface" is too much here? 

> `src/assets/i18n/en.json` (and all other language files) should be updated too, shouldn't it, with the new string to translate?

Yes. I'd like to get the proposal validated before going into the translation.

Reply:
> > Isn't "Change language" enough? Or "Select language"?
> 
> This is to explain the purpose of the field, I think it's better to be specific. Could you expand why you think "interface" is too much here?

Yes, I see that, but in this context "interface" is not needed. I will be quite obvious that "change language" will change the language the result is presented in. Shorter is better. Furthermore, "interface" is a quite technical term that can distract users.